client/python/openlineage/client/run.py

```diff
@@ -10,6 +10,8 @@
 from openlineage.client.facet import NominalTimeRunFacet, ParentRunFacet
 from openlineage.client.utils import RedactMixin
 
+SCHEMA_URL = "https://openlineage.io/spec/1-0-5/OpenLineage.json#/definitions/RunEvent"
```

Comment:
will the be kept in sync with the version in https://github.com/OpenLineage/OpenLineage/blob/0926f98b65c019b5003f7a19f55a7daaa0ee869c/spec/OpenLineage.json#L3 ?

Reply:
I'd imagine so 🤔 though the python client might trail behind the JSON I'd imagine unless we enforce it otherwise in PRs.

Reply:
The medium-term plan is generating the types from JSON Schema, instead of manually creating them.